### PR TITLE
A couple missing rust commands

### PR DESCRIPTION
**Pull request by RyanRuane**

### CODE REVIEW

File: code/create_spoken_forms.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import itertools
 import re
 from collections import defaultdict
+from collections.abc import Mapping
```
Comment (rntz) [inline code/create_spoken_forms.py:4]: What's up with these changes from typing to collections.abc?

**Review by rntz — APPROVED** [review]
Looks okay. What's the purpose of changing from typing to collections.abc though?

File: code/create_spoken_forms.py (hunk 1 above)
Comment (RyanRuane) [reply code/create_spoken_forms.py:4]: good question. I think my branch may have been out of sync and not up to date. I can put it back.

Comment (pokey) [reply code/create_spoken_forms.py:4]: Our autoformatter started doing that last week.  Apparently as of 3.9 they're starting [to deprecate](https://peps.python.org/pep-0585/#:~:text=Importing%20those%20from%20typing%20is%20deprecated) the versions from `typing` in favour of those from `collections.abc`

File: lang/rust/rust.talon [hunk 2]
```diff
@@ -48,6 +48,7 @@ toggle use: user.code_toggle_libraries()
 ## Simple aliases
 borrow: "&"
 borrow mutable: "&mut "
+state (async | asynchronous): "async "
```
Comment (pokey) [inline lang/rust/rust.talon:51]: I use "a sink" for this, as I found recognition better.  Not sure our stance regarding these kind of games to improve recognition tho.  cc/ @knausj85 @lunixbochs

File: code/create_spoken_forms.py (hunk 1 above)
Comment (auscompgeek) [reply code/create_spoken_forms.py:4]: Yeah, I think this is coming from a new [pyupgrade](https://github.com/asottile/pyupgrade) release.

Comment (auscompgeek) [reply code/create_spoken_forms.py:4]: Looks like this change was made in #921, so if you rebase this change should disappear.

File: lang/rust/rust.talon (hunk 2 above)
Comment (auscompgeek) [reply lang/rust/rust.talon:51]: At some point it might be worth adding this under a common tag to make this consistent between languages.

Comment (pokey) [reply lang/rust/rust.talon:51]: Fwiw we support this [as a keyword](https://github.com/knausj85/knausj_talon/blob/3eab2c9525c3896718152c1f88e0f2d0ceee1237/lang/javascript/javascript.py#L48) in Javascript

**Comment by RyanRuane** [comment]
rebased

**Review by auscompgeek — APPROVED** [review]

**Pull request merged**